README.md

```diff
@@ -1,27 +1,12 @@
 # provider-crossplane
 
-# Overview
-The purpose of this repository is to manage and integrate some infrastructure resources in AWS leveraging on [Crossplane](https://github.com/crossplane/crossplane). It's currently in an early development stage. 
+## What is Provider Crossplane?
+Provider Crossplane is a operator designed to complement the [Kops Operator](https://github.com/topfreegames/kubernetes-kops-operator) by providing a way to manage infrastructure resources in AWS leveraging on [Crossplane](https://github.com/crossplane/crossplane).
```

Comment:
To complement initially kops-operator and support other providers in the future

Reply:
Also, it is important to add a reference to the cluster-api project, because the clustermesh for example, uses the `clusters.cluster.x-k8s.io` which is part of the cluster-api and not kops-operator or other providers

Reply:
WDYT?

Reply:
LGTM